src/routes/safe/store/selectors/newTransactions.ts

```diff
@@ -0,0 +1,37 @@
+import { NewTransactionsState, TRANSACTIONS } from '../reducer/newTransactions'
```

Comment:
I'm not sure these selectors are needed at all, I mean they are not deriving any data. 
Perhaps we can omit this indirection to obtain the data for now.

Reply:
They let you take the transactions for the current page

Reply:
but we will not have the concept of "page", once we add them to the table, there is not `goToPreviousPage()`. If the user scrolls up, we will already have loaded the rows.

Reply:
I'm not 100% sure how we are gonna implement this, as far I discussed with @fernandomg , would be nice if we can store like one page before, the current page and one page after, and every time we scroll down we update the stored pages on memory, example:

Pages on memory: [1,2,3] -> user is at the beginning of the scroll
Once the users start scrolling down if we implement some sort of mechanism that allows us now where is him, we could now that for instance, he is almost or in page 3 and start fetching data for the page 4 before he ends the scrolling, in that case those selectors would be useful

Meanwhile, I just suggest to leave this at it is and remove if we really need to remove it once we implement the infinite scroll